app/models/project.rb

```diff
@@ -91,6 +91,17 @@ class Project < ApplicationRecord
 
   scope :by_language, lambda { |languages|
     joins(:language).where(languages: { name: languages })
+  scope :with_no_cc, lambda {
```

Comment:
maybe `without` reads better than `with_no` and is also more consistent with some other places